### ISSUE TITLE
Error when importing moment from TypeScript

### ISSUE THREAD

**Issue by acerix** [fullcalendar/fullcalendar#4691]
Reported in https://github.com/fullcalendar/fullcalendar/issues/4680#issuecomment-493017560

Steps to reproduce:

Clone and build typescript example: https://github.com/fullcalendar/scheduler-typescript-example

Include moment plugin as in this example: https://fullcalendar.io/docs/moment-plugins

Error on `npm run build`:
```
ERROR in ./src/example.ts
Module not found: Error: Can't resolve '@fullcalendar/moment' in '/home/user/scheduler-typescript-example/src'
 @ ./src/example.ts 7:0-48 11:104-116

ERROR in /home/user/scheduler-typescript-example/src/example.ts
./src/example.ts
[tsl] ERROR in /home/user/scheduler-typescript-example/src/example.ts(7,26)
      TS2307: Cannot find module '@fullcalendar/moment'.
```

**Comment by trungk18** [comment]
Hi, I will issue a pull request for this one. It is really quick to fix.

**Comment by trungk18** [comment]
I found the workaround for that. I added this line into `tsconfig.json` file and now my project got built successfully. 

```
"allowSyntheticDefaultImports": true
```

Refer to official moment.js [doc](http://momentjs.com/docs/#/use-it/typescript/)

**Comment by acerix** [comment]
Thanks for the info, was mentioned in another issue

https://github.com/fullcalendar/fullcalendar/issues/4580

I'll leave this open so it can be added to the docs.

**Comment by arshaw** [comment]
fixed in master. will release soon

(won't need allowSyntheticDefaultImports)

**Comment by arshaw** [comment]
This was fixed in 4.3.0 though please start using 4.3.1
Releases: https://github.com/fullcalendar/fullcalendar/releases

No need for `allowSyntheticDefaultImports` anymore. FullCalendar's ts def file now imports moment in a non-synthetic way
